resources/aws/vpc.go

```diff
@@ -161,3 +166,32 @@ func (v VPC) GetID() (string, error) {
 
 	return *vpc.VpcId, nil
 }
+
+// List lists the VPCs for this installation.
+func (v VPC) List() ([]VPC, error) {
+	vpcs := []VPC{}
+
+	resp, err := v.Clients.EC2.DescribeVpcs(&ec2.DescribeVpcsInput{
+		Filters: []*ec2.Filter{
+			{
+				Name: aws.String(fmt.Sprintf("tag:%s", tagKeyInstallation)),
+				Values: []*string{
+					aws.String(v.InstallationName),
+				},
+			},
+		},
+	})
+	if err != nil {
+		return []VPC{}, microerror.Mask(err)
+	}
+
+	for _, vpc := range resp.Vpcs {
```

Comment:
Why do you tag vpcs with the installation name after fetching them?

Reply:
We use the cluster ID as the Name tag for VPCs. So this is getting the cluster IDs rather than tagging with the installation name.

Reply:
Ah, I get you. Reading is hard.